src/features/submitCode/gitPush/ui/GitPushDialog.tsx

```
@@ -13,13 +13,14 @@ import { Switch } from '@/components/ui/switch';
 import { Tooltip, TooltipContent, TooltipTrigger } from '@/components/ui/tooltip';
 import { useEffect, useState } from 'react';
 import { Select } from '@/shared/ui/select/Select';
+import { useUserStore } from '@/entities/user/model/store';
 
 export default function GitPushDialog({}) {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)